src/filesystem/filesystem_helpers.cpp

```diff
@@ -0,0 +1,10 @@
+#include "nova_renderer/filesystem/filesystem_helpers.hpp"
+
+#include <stdarg.h>
+#include <string.h>
+
+namespace nova::filesystem {
+    void copy_path_parts_to_buffer(const rx_size offset, char* buffer, const rx::string& path_part) {
```

Comment:
Does this need to check if the buffer is large enough?

Reply:
Deleted the function